CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [ENHANCEMENT] Ingester/Query-Frontend: Avoid using automatic private IPs (APIPA) upon assignment. APIPA still used as last resort with logging indicating usage. #4032
```

Comment:
@jbvmio Would you mind moving this entry down so it is above the other `ENHANCEMENT` entries?

```suggestion
* [ENHANCEMENT] Ingester/Query-Frontend: Avoid using automatic private IPs (APIPA) upon assignment. APIPA still used as last resort with logging indicating usage. #4032
```

Reply:
No problem, done.